src/MarkDown/About.md

```diff
@@ -0,0 +1,9 @@
+# About
+
+MovieMentor is an application developed by Jeffrey DiVincent, Adam Fernandes, Jacob Franz, Nathanael Gaulke, Lauren Morris, and Garrett Spears as part of our Senior Design Project for Computer Science at the University of Central Florida. 
+
+We wanted to create a place where users could gain more detailed information on potential trauma and content sensitivities found in certain movies. The MPAA rating system has a general guideline for ages and content found in movies, but has been subject to differing classification over the years making it merely a starting point. We have also used sites like Common Sense Media and Does the Dog Die, however we found that they focused on parental guidance and extremely traumatic events. We wanted to have a more wider platform that accommodated all sorts of potentially sensitive issues to serve a wider audience and include other groups of trauma that may be overlooked by the general population. Additionally, we wanted to designed the platform so that a user could hide content warnings they do not want to come in contact with at all, as well as reduce the chance of movie spoilers by having users opt to look closer at a content warning they might be on the fence about. 
```

Comment:
should be "design" not "designed" here or could be changed to "We ~~wanted to~~ designed the platform..."